build.sbt

```diff
@@ -183,7 +192,7 @@ lazy val keytool = (project in file("keytool"))
     buildInfoPackage := "org.constellation.keytool",
     buildInfoOptions ++= Seq(BuildInfoOption.BuildTime, BuildInfoOption.ToMap),
     mainClass := Some("org.constellation.keytool.KeyTool"),
-    libraryDependencies ++= keyToolSharedDependencies
+    libraryDependencies ++= (keyToolSharedDependencies ++ testDependencies ++ fs2Dependencies)
```

Comment:
Why keytool needs test and fs2 dependencies?

Reply:
Good catch 👍  It's a leftover from draft changes to keytool.